src/tools/android/java/com/google/devtools/build/android/r8/BUILD

```diff
@@ -41,6 +41,8 @@ java_library(
     }),
     visibility = ["//src/test/java/com/google/devtools/build/android/r8:__pkg__"],
     deps = [
+        "//src/main/java/com/google/devtools/build/lib/worker:work_request_handlers",
+        "//src/main/java/com/google/devtools/build/lib/worker:worker",
```

Comment:
Do you actually need to depend on `:worker`? That should only be for internal (server-side) use.